stories/Templates/ProjectPage/ProjectPage.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect } from "react";
+import viewport from "../../assets/js/viewport";
+import { PageHero } from '../../Components/UIcomponents/Hero/Pagehero/PageHero';
+import CountrySiteHeader from "../../Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader";
+import { HeadingBig } from "../../Molecules/Text/HeadingBig/HeadingBig";
+import { Footer } from "../../Organism/Footer/Footer";
+import { Heading } from "../../Atom/Typography/Heading/Heading";
+import { P } from '../../Atom/Base-typography/Paragraph/Paragraph';
+import { DownloadCard } from '../../Molecules/Blocks/DownloadCard/DownloadCard';
+import { Projectdetail } from '../../Organism/Text/ProjectDetail/ProjectDetail';
+import { CtaButton } from '../../Components/UIcomponents/Buttons/Cta_button/CtaButton';
+import { ContentCardWithAndWithoutImage } from '../../Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage';
+import { Icondownload } from '../../Atom/Icons/download';
+import { Statscardslider } from '../../Organism/Statscardslider/Statscardslider';
+import "./projectpage.scss";
```

Comment:
@neelesh-tippe Kindly use single quotes `'` in the import statements. You can read more about here:
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings